modules/consensus/accept.go

```diff
@@ -120,11 +120,12 @@ func (cs *ConsensusSet) addBlockToTree(b types.Block) (ce changeEntry, err error
 	return ce, nil
 }
 
-// AcceptBlock will add a block to the state, forking the blockchain if it is
-// on a fork that is heavier than the current fork. If the block is accepted,
-// it will be relayed to connected peers. This function should only be called
-// for new, untrusted blocks.
-func (cs *ConsensusSet) AcceptBlock(b types.Block) error {
+// managedAcceptBlock accepts a block but does not broadcast it to any peers.
+// See comment for AcceptBlock. Typically AcceptBlock should be used. This
+// method should only be used when there would otherwise be multiple
+// consecutive calls to AcceptBlock with each successive call accepting the
+// child block of the previous call.
+func (cs *ConsensusSet) managedAcceptBlock(b types.Block) error {
```

Comment:
the docstring here is a little weird, because `managedAcceptBlock` is not an exported function, and it gets called by `AcceptBlock`.  I would probably copy the `AcceptBlock` docstring and then remove the part about the broadcast, and the part about only being called on new blocks.